### ISSUE TITLE
XSR-SIM does not work after computer restart / sleep / hibernate

### ISSUE THREAD

**Issue by azaz44** [FrSkyRC/RF_Feedback_Group#62]
I have two of them (FrSky XSR-SIM) and both have same problem, with newest firmware. Tested on different PCs. I asked the question in a Facebook group and more people confirmed this to be a problem for them.

I'm running Windows 10. I don't know if problem is Windows-specific.

Symptoms are:
- it happens after computer restart / power on / sleep / hibernate. Not 100% of the time but almost.
- in such situation, turning on the radio (so it's transmitting) makes XSR-SIM show green light. But game controller in Windows gets no movements.
- it can be easily confirmed using this window:
![image](https://github.com/user-attachments/assets/5ba9c11a-99e5-49f2-9387-7983f77d2690)
- need to unplug XSR-SIM and plug it in again, to resolve the problem

So basically, we need to unplug the dongle and plug it in again after every computer sleep or restart.

For me, the problem always existed, even when XSR-SIM was shipped with older (unencrypted) ACCST D16. So it is not anything introduced lately. Updates to newer D16 didn't change anything in this regard.

Can software update fix it?

**Comment by azaz44** [comment]
> * on the radio (so it's transmitting) makes XSR-SIM show green

Is it a software issue, or hardware?
to be honest, I would spend money on a third XSR-SIM if this would be fixed in hardware. The problem is just so annoying, I replug it several times per day.
